ci_framework/roles/ci_setup/defaults/main.yml

```diff
@@ -20,3 +20,4 @@
 
 cifmw_ci_setup_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}"
 cifmw_ci_setup_openshift_client_version: "stable"
+cifmw_ci_setup_oc_install_path: "{{ ansible_user_dir ~ '/bin' }}"
```

Comment:
shall we find a way to inject this in the `cifmw_path` ? FTR, this parameter is set in 01_bootstrap.yml playbook (and, sadly, in some scenario files)

Reply:
Should be nice, I'll give it a thought